BUILD.md

```diff
@@ -0,0 +1,170 @@
+# Build FreeOrion
+
+
+## Hardware and OS Requirements
+
+FreeOrion will compile on most recent Windows, Mac OS X and Linux operating
+systems. Other operating systems have reported to work, but support is not
+actively maintained by the FreeOrion developers. FreeOrion is developed for x86
+compatible processor architectures, other architectures haven't been tested.
+
+The FreeOrion source code makes heavy use of templates and requires much memory
+to compile ; 4GB RAM or more is recommended. To build FreeOrion, expect up to
+45 minutes on a Intel Core i5 system.
+
+
+## Required Software Dependencies
+
+FreeOrion depends on the following software to build:
+
+  * [Visual Studio] - 2015 for Windows Desktop Update 3 ; Windows only
+  * [Xcode] - 6.4 or later ; Mac OS X only
+  * [CMake] - 3.4 (Mac OS X) ; 3.1 or later (Other non-Windows)
+  * A C++11 compliant compiler - Other Operating Systems
+    * [GNU GCC] - 5.0 or later
+    * [Clang] - 3.9 or later
+  * [Python] - 2.7 or later, but not 3 or later
```

Comment:
I personally think this line would read better as 
```
[Python] - 2.7.x (not 3.x)
```

Reply:
Your suggestion looks out of place compare to the pattern used in the previous entries, what about:

```
  * [Python] - 2.7 or later (not 3)
```

Reply:
[There will be no Python 2.8](https://www.python.org/dev/peps/pep-0404/), so requiring specificically 2.7.x seems most precise.

Reply:
> Your suggestion looks out of place compare to the pattern used in the previous entries

It is indeed different than the previous patterns, precisely because of the basic difference in the requirement.  It is the first "or later", the one about 2.7, which is incomplete/misleading, so removing the second one doesn't really help.  Taken as a whole, either of your formulations do wind up accurately expressing the requirement, but it seems to me pointless and detrimental to leave an inconsistency/inaccuracy in the middle of them.

Reply:
Fixed